src/server-entry.ts

```diff
@@ -1,34 +1,45 @@
 import buildApp from '@/common/buildApp';
 import { config } from './server/TermboxConfig';
 import { factory } from './common/TermboxFactory';
-import WikibaseApiLanguageRepository from './server/data-access/WikibaseApiLanguageRepository';
+import WikibaseContentLanguagesRepo from './server/data-access/WikibaseContentLanguagesRepo';
 import MwBotWikibaseRepo from './server/data-access/MwBotWikibaseRepo';
 import mwbot from 'mwbot';
 import EntityInitializer from './common/EntityInitializer';
 import BundleBoundaryPassingException, { ErrorReason } from '@/common/exceptions/BundleBoundaryPassingException';
 import TermboxRequest from '@/common/TermboxRequest';
 import EntityNotFound from '@/common/data-access/error/EntityNotFound';
+import MwBotWikibaseContentLanguagesRepo from './server/data-access/MwBotWikibaseContentLanguagesRepo';
+import TranslationLanguageNotFound from './common/data-access/error/TranslationLanguageNotFound';
 
-const apiBot = new mwbot( {
-	apiUrl: config.getWikibaseRepoApi(), // TODO test
-} );
+export default ( termboxRequest: TermboxRequest ) => {
+	const apiBot = new mwbot( {
+		apiUrl: config.getWikibaseRepoApi(), // TODO test
+	} );
 
-factory.setLanguageTranslationRepository(
-	new WikibaseApiLanguageRepository(
-		apiBot,
-	),
-);
-factory.setEntityRepository(
-	new MwBotWikibaseRepo(
-		apiBot,
-		new EntityInitializer(),
-	),
-);
+	const languageRepo = new WikibaseContentLanguagesRepo(
```

Comment:
Had we continued doing this outside the export, all requests would have a shared the same language API response. This could be, carefully crafted, a performance optimization in the future but should be an active decision. (Not currently possible for this scenario with the API response containing request-specific content [translation])